<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ssrtools</class>
 <widget class="QWidget" name="ssrtools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="m_toolButton_options">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>option</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pushButton_start">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="m_radioButton_fullscreen">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>96</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>fullscreen</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="m_radioButton_fixed">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>96</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>fixed</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="m_radioButton_cursor">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>96</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>cursor</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_label_video_x">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>55</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_label_video_y">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>55</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Top</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_label_video_width">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>55</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Width</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_label_video_h">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>55</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Height</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>VideoFrate</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spinbox_video_x">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>42</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>2000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spinbox_video_y">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>42</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spinbox_video_w">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>42</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_spinbox_video_h">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>42</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pushbutton_video_select_rectangle">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>210</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>select rectangle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pushbutton_video_select_window">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>select window</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
